superset-frontend/src/SqlLab/components/ResultSet.tsx

```diff
@@ -42,6 +43,28 @@ import { Query } from '../types';
 
 const SEARCH_HEIGHT = 46;
 
+const debounce = (
+  func: { apply: (arg0: any, arg1: IArguments) => void },
+  wait: number,
+  immediate: number,
+) => {
+  let timeout: NodeJS.Timeout | null;
+  return function () {
+    // eslint-disable-next-line @typescript-eslint/no-this-alias
+    const context = this;
+    // eslint-disable-next-line prefer-rest-params
+    const args = arguments;
+    const later = function () {
```

Comment:
have you tried an arrow function here so that you don't need to assign `this` to context?

Reply:
actually, nm, if you use lodash, you won't need this.